src/cli/test/logger_test.py

```diff
@@ -12,7 +12,7 @@
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
-openstudio.Logger_instance().standardOutLogger().setLogLevel(openstudio.Error)
+openstudio.Logger.instance().standardOutLogger().setLogLevel(openstudio.Error)
```

Comment:
Swig bump cause the openstudio.Logger_instance() to be gone. `openstudio.Logger.instance()` is what I should have used to begin with.